bluemira/geometry/tools.py

```diff
@@ -983,6 +983,39 @@ def sweep_shape(
     return convert(result, label=label)
 
 
+def loft_shape(
+    profiles: Iterable[BluemiraWire],
+    *,
+    solid: bool = True,
+    ruled: bool = False,
+    label: str = "",
+) -> BluemiraSolid | BluemiraShell:
+    """
+    Loft between a set of profiles.
+
+    Parameters
+    ----------
+    profiles:
+        Profile(s) to loft between
+    solid:
+        Whether or not to create a Solid
+    ruled:
+        created a ruled shape
```

Comment:
```suggestion
        Create a ruled shape
```

Would also be useful to know briefly what a ruled shape is

Reply:
I'll just put this in the docs, it'll be better than my description https://en.wikipedia.org/wiki/Ruled_surface